{"uuid": "27ee64eb-4f71-4b6b-b117-e3a6ba3a5bb5", "children": ["96d88517-b33f-4c49-93fd-24bed531c600", "ee442d1d-df5c-4503-b153-55b8f6d44e24"], "befores": [{"name": "_Class__pytest_setup_class", "status": "passed", "start": 1596043428932, "stop": 1596043428932}], "afters": [{"name": "_Class__pytest_setup_class::0", "status": "broken", "statusDetails": {"message": "AttributeError: type object 'TestCase' has no attribute 'driver'\n", "trace": "  File \"D:\\python\\lib\\site-packages\\allure_commons\\_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"D:\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 800, in _teardown_yield_fixture\n    next(it)\n  File \"D:\\python\\lib\\site-packages\\_pytest\\python.py\", line 705, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"D:\\python\\lib\\site-packages\\_pytest\\python.py\", line 632, in _call_with_optional_argument\n    func(arg)\n  File \"D:\\appium_PO_kaoyan\\test_case\\test_cases.py\", line 23, in teardown_class\n    Common(cls.driver).send_email('../report/pytest_report.html')\n"}, "start": 1596043470893, "stop": 1596043470896}], "start": 1596043428932, "stop": 1596043470896}